hyperion/app/main.py

```
@@ -44,89 +46,175 @@
+                [(private_key, certificate_chain)],
+                root_certificates=root_certificates,
+                require_client_auth=bool(root_certificates),
+            )
+
+            # Add secure port
+            self.server.add_secure_port(settings.GRPC_ADDRESS, server_credentials)
+            logger.info(f"TLS enabled with certificate: {settings.TLS_CERT_PATH}")
+
+        except Exception as e:
+            logger.error(f"Failed to configure TLS: {e}")
+            raise RuntimeError(f"TLS configuration failed: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)